packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx

```diff
@@ -276,7 +277,24 @@ export const AddObjectMenu = (): JSX.Element => {
           uri: modelUri,
           modelType: modelType ?? ext.toUpperCase(),
         };
-
+        if (ext.toUpperCase() == 'GLTF') {
+          const animationComponent: IAnimationComponent = {
+            currentAnimations: [],
+            type: 'Animation',
+            uri: modelUri,
+          };
+          const node = {
+            name: filename,
+            components: [gltfComponent, animationComponent],
+            parentRef: mustBeRoot ? undefined : getRefForParenting(),
+          } as unknown as ISceneNodeInternal;
```

Comment:
why cast to unknown? because of no `ref`? you should use type `ISceneNode` instead